backend/dissemination/views.py

```diff
@@ -225,10 +279,35 @@ def get(self, request, report_id, file_type):
         return redirect(get_download_url(filename))
 
 
-class SummaryReportDownloadView(ReportAccessRequiredMixin, View):
+class SingleSummaryReportDownloadView(ReportAccessRequiredMixin, View):
     def get(self, request, report_id):
+        """
+        Given a report_id in the URL, generate the summary report in S3 and
+        redirect to its download link.
+        """
         sac = get_object_or_404(General, report_id=report_id)
         filename = generate_summary_report([sac.report_id])
         download_url = get_download_url(filename)
 
         return redirect(download_url)
+
+
+class MultipleSummaryReportDownloadView(View):
```

Comment:
I think it'd be good to have a couple of basic tests around this View - don't think we need to go so far as to inspect the resultant xlsx file in the test, but we should have some coverage around the various cases (what should this view return if no reports are found, some reports but less than the max, more than the max, etc)